src/custom/components/WalletModal/InjectedOption.tsx

```diff
@@ -0,0 +1,53 @@
+import { Trans } from '@lingui/macro'
```

Comment:
Is there anything modded here?

Reply:
It should be https://github.com/cowprotocol/cowswap/blob/uni-merge/1.0.7/src/custom/components/WalletModal/InjectedOption.tsx